ratpack-core/src/main/java/ratpack/http/Request.java

```diff
@@ -130,6 +130,12 @@
   HostAndPort getLocalAddress();
 
   /**
+   * A flag representing whether or not the request originated via AJAX.
+   * @return A flag representing whether or not the request originated via AJAX.
+   */
+  Boolean isAjaxRequest();
```

Comment:
If it's not meaningful for this to be `null`, might want to change this to return `boolean`.